src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -1025,7 +1025,7 @@ $(document).ready(() => {
 
             // disable button to prevent user from clicking submission button again
             const $submissionButton = $('#response_submit_button');
-            addLoadingIndicator($submissionButton, '');
+            addLoadingIndicator($submissionButton, 'Submitting ');
```

Comment:
I wonder why the `...` isn't showing up in my PC. Ignore my previous comment.

_Sent from my Motorola Moto G Play using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_